docs/src/development/troubleshoot.md

```diff
@@ -118,6 +118,17 @@ You can write to disk a `build` hook to generate anything you need later.
 Or you can declare writable [mounts](../configuration/app/storage.md#mounts), which are writable even during and after deploy.
 They can be used for your data: file uploads, logs, and temporary files.
 
+## Git push fails due to lack of disk space
+
+If you see the following message when attempting to do a `git push`:
+`There is not enough free space to complete the push`
+
+It usually indicates that large files are present in the repository (where they shouldn't be).
+Make sure that the paths for media files, dependencies, or databases, ... are set to be ignored in the `.gitignore` file.
+
+If large files are already in the repository, the open-source [bfg-repo-cleaner](https://rtyley.github.io/bfg-repo-cleaner/) can assist in cleaning up the repository by purging older commits, removing unnecessary files, ...
+Create a [support ticket](https://console.platform.sh/-/users/~/tickets/open) to have the disk space temporarily increased.
```

Comment:
```suggestion
To have the disk space temporarily increased, create a [support ticket](https://console.platform.sh/-/users/~/tickets/open).
```

This suggestion is about the general [clause order recommendations]. I'd also consider starting with some more context (like "if this doesn't work" or something).

Also, would it only be for a temporary increase? I thought it wasn't temporary. If it's only temporary, it seems like we should push the other fixes more?

Reply:
The temporality of the fix depends on the size that is required. As far as I know, some sizes can not be set as a global fix (but I might be wrong). Anyway, repositories that have sizes over 4-8 GB are probable signs of issues in development. 
For e.g [limit in Github is 5GB](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)

Reply:
The fact that its a sign of an issue seems to me a sign that we shouldn't be suggesting a temporary fix then. If it's a development issue, then that should be fixed. Otherwise, it's not a temporary fix: it's intentional and should be introduced differently.

Reply:
You are correct, I removed that part out of the documentation. It usually is a sign of a development issue, and the courtesy that support allows is probably not always a good approach.